guides/pages.yml

```
@@ -249,6 +249,50 @@
   pages:
     - title: "Gérer les Dépendances"
       url: "index"
+- title: "Utiliser TypeScript"
+  url: "typescript"
+  pages:
+    - title: "Utiliser TypeScript avec Ember"
+      url: "index"
+    - title: "TypeScript: pour bien démarrer"
+      url: "getting-started"
+    - title: "TypeScript: concepts fondamentaux"
+      url: "core-concepts"
+      pages:
+        - title: "TypeScript et Ember"
+          url: "index"
+        - title: "TypeScript: invokables"
+          url: "invokables"
+        - title: "TypeScript: routing"
+          url: "routing"
+        - title: "TypeScript: services"
+          url: "services"
+        - title: "TypeScript et EmberData"
+          url: "ember-data"
+    - title: "TypeScript: développement d'une application"
+      url: "application-development"
+      pages:
+        - title: "Introduction"
+          url: "index"
+        - title: "TypeScript: configuration"
+          url: "configuration"
+        - title: "TypeScript: tests"
+          url: "testing"
+        - title: "TypeScript: convertir une app Ember existante"
+          url: "converting-an-app"
+        - title: "TypeScript: addons"
```

In English `addon` and `add-on` may be used, to me in French only `add-on` is used (even if it's an Anglicism).

```suggestion
        - title: "TypeScript: add-ons"
```